code/game/response_team.dm

```diff
@@ -57,76 +55,26 @@ var/can_call_ert
 			return
 		if(jobban_isbanned(usr, MODE_ERT) || jobban_isbanned(usr, "Security Officer"))
-			to_chat(usr, SPAN_DANGER("You are jobbanned from the emergency reponse team!"))
+			to_chat(usr, SPAN_DANGER("You are jobbanned from the MTF!"))
 			return
 		if(GLOB.ert.current_antagonists.len >= GLOB.ert.hard_cap)
-			to_chat(usr, "The emergency response team is already full!")
+			to_chat(usr, "The MTF is already full!")
 			return
 		GLOB.ert.create_default(usr)
 	else
 		to_chat(usr, "You need to be an observer or new player to use this.")
 
-// returns a number of dead players in %
-/proc/percentage_dead()
-	var/total = 0
-	var/deadcount = 0
-	for(var/mob/living/carbon/human/H in SSmobs.mob_list)
-		if(H.client) // Monkeys and mice don't have a client, amirite?
-			if(H.stat == 2) deadcount++
-			total++
-
-	if(total == 0) return 0
-	else return round(100 * deadcount / total)
-
-// counts the number of antagonists in %
-/proc/percentage_antagonists()
-	var/total = 0
-	var/antagonists = 0
-	for(var/mob/living/carbon/human/H in SSmobs.mob_list)
-		if(is_special_character(H) >= 1)
-			antagonists++
-		total++
-
-	if(total == 0) return 0
-	else return round(100 * antagonists / total)
-
-// Increments the ERT chance automatically, so that the later it is in the round,
-// the more likely an ERT is to be able to be called.
-/proc/increment_ert_chance()
-	while(send_emergency_team == 0) // There is no ERT at the time.
-		var/decl/security_state/security_state = decls_repository.get_decl(GLOB.using_map.security_state)
-		var/index = list_find(security_state.all_security_levels, security_state.current_security_level)
-		ert_base_chance += index
-		sleep(600 * 3) // Minute * Number of Minutes
-
-
-/proc/trigger_armed_response_team(force = 0, reason = "")
-	if(!can_call_ert && !force)
-		return
+/proc/trigger_armed_response_team(reason = "")
 	if(send_emergency_team)
 		return
 
-	var/send_team_chance = ert_base_chance // Is incremented by increment_ert_chance.
-	send_team_chance += 2*percentage_dead() // the more people are dead, the higher the chance
-	send_team_chance += percentage_antagonists() // the more antagonists, the higher the chance
-	send_team_chance = min(send_team_chance, 100)
-
-	if(force) send_team_chance = 100
-
-	// there's only a certain chance a team will be sent
-	if(!prob(send_team_chance))
-		command_announcement.Announce("It would appear that an emergency response team was requested for [station_name()]. Unfortunately, we were unable to send one at this time.", "[GLOB.using_map.boss_name]")
-		can_call_ert = 0 // Only one call per round, ladies.
-		return
-
-	command_announcement.Announce("It would appear that an emergency response team was requested for [station_name()]. We will prepare and send one as soon as possible.", "[GLOB.using_map.boss_name]")
+	command_announcement.Announce("It would appear that an MTF was requested for [station_name()]. We will prepare and send one as soon as possible.", "[GLOB.using_map.boss_name]")
```

Comment:
```suggestion
	command_announcement.Announce("It would appear that MTF was requested for [station_name()]. We will prepare and send one as soon as possible.", "[GLOB.using_map.boss_name]")
```

Reply:
"It would appear that Mobile Task Force was requested for..." sounds pretty stupid, the 'an' is grammatically correct here.

Reply:
Wouldn't it be a then and not an

Reply:
> Wouldn't it be a then and not an

[When choosing between A and AN for an acronym you use the pronunciation of the acronym, not of the full phrase.](https://editorsmanual.com/articles/indefinite-article-a-an-with-abbreviations/)

Reply:
> > Wouldn't it be a then and not an
> 
> [When choosing between A and AN for an acronym you use the pronunciation of the acronym, not of the full phrase.](https://editorsmanual.com/articles/indefinite-article-a-an-with-abbreviations/)

eat a pickle

Reply:
> > > Wouldn't it be a then and not an
> > 
> > 
> > [When choosing between A and AN for an acronym you use the pronunciation of the acronym, not of the full phrase.](https://editorsmanual.com/articles/indefinite-article-a-an-with-abbreviations/)
> 
> eat a pickle

I'll get on that STAT mr president!